src/software/ai/hl/stp/play/BUILD

```
@@ -555,7 +555,7 @@ py_test(
 )
 
 cc_test(
-    name = "shoot_or_chip_play_test",
+    name = "shoot_or_chip_play_test_old",
```

nit: Just to stay consistent
```suggestion
    name = "shoot_or_chip_play_cpp_test",
```